tests/acceptance/TestHelpers/GraphHelper.php

```diff
@@ -43,6 +43,7 @@ class GraphHelper {
 	public const ADDITIONAL_PERMISSIONS_ROLES = [
 		'Secure Viewer' => 'aa97fe03-7980-45ac-9e50-b325749fd7e6',
 		'Space Editor Without Versions' => '3284f2d5-0070-4ad8-ac40-c247f7c1fb27',
+		'Denied' => '63e64e19-8d43-42ec-a738-2b6af2610efa',
```

Comment:
This has already been added in PR https://github.com/owncloud/ocis/pull/10722. Rebase this PR. :+1: